shared/params_test.go

```diff
@@ -344,6 +337,18 @@ func TestParseProductSpec(t *testing.T) {
 	assert.Equal(t, "edge", productSpec.BrowserName)
 }
 
+func TestParseProductSpec_FullSHA(t *testing.T) {
+	sha := "0123456789aaaaabbbbbcccccdddddeeeeefffff"
+	r := httptest.NewRequest("GET", "http://wpt.fyi/?product=chrome@"+sha, nil)
+	filters, err := ParseTestRunFilterParams(r)
+	assert.Nil(t, err)
+	products := filters.GetProductsOrDefault()
+	assert.Len(t, products, 1)
+	if len(products) > 0 {
```

Comment:
Isn't this condition redundant given the above assertion? Or does the test keep running on failed asserts?

Reply:
Unfortunately no, Golang keeps going on a failed assert.

Reply:
Oh, weird :)